src/components/Dropdown/Presentation.tsx

```diff
@@ -0,0 +1,46 @@
+import { Button, ListItem, UnorderedList } from "@chakra-ui/react";
+
+import { Dispatch, SetStateAction } from "react";
+
+import styles from "./dropdown.module.css";
+import { DROPDOWN_LINKS, LOGOUT_API } from "./DropdownConstants";
+import Link from "next/link";
+import { setIsLoggedOut } from "@/src/store/global";
+import { useDispatch } from "react-redux";
+
+export default function Dropdown({ setIsDropdownVisible } : {
+    setIsDropdownVisible: Dispatch<SetStateAction<boolean>>;
+}) {
+    const reduxDispatch = useDispatch();
+
+    const dropdownItems = DROPDOWN_LINKS.map((link) => {
+        return (
+            <ListItem key={link.id} className={styles.dropdown_items}>
+                <Link href={link.link} className={styles.dropdown_links}>
+                    {link.name}
+                </Link>
+            </ListItem>
+        )
+    })
+
+    const logout = () => {
+        fetch(LOGOUT_API, {
+            method: 'GET',
+            credentials: 'include',
+        }).then(() => {
+            reduxDispatch(setIsLoggedOut({isLoggedIn: false}))
+        });
+    }
+
```

Comment:
can we separate the UI from the logic, let's create a container and presentation component as we are following in all the sites

Reply:
Yes and thanks for explaining the difference between presentation component and container